Gemfile

```diff
@@ -59,7 +59,7 @@ group :development, :test do
   gem 'rubocop', '~> 0.42.0', require: false
 
   # Security scanners
-  gem 'brakeman'
+  gem 'brakeman', '3.4.1'
```

Comment:
I think the standard here is to not lock the version in the Gemfile unless there's a specific reason to do so.